### PR TITLE
Squash down Spark/Toil jobs

### LINKED ISSUE #134: Restructure ADAM pipeline to have a single monolithic Toil job
That runs all the Spark jobs. This is because the Spark jobs do not write to the file store and thus should not be considered atomic as is written now.

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #134. Also, I cleaned up some logging funk.

### CODE REVIEW

**Comment by heuermh** [comment]
LGTM.  Also fixes #110.

**Comment by fnothaft** [comment]
Thanks for catching that @heuermh! I've amended the second commit message to mark that it resolves #110.

File: src/toil_scripts/adam_pipeline/spark_toil_script.py [hunk 1]
```diff
@@ -59,11 +60,11 @@ def start_workers(job, masterIP, inputs):
     """
     Starts the worker services.
     """
-    log.write("workers job\n")
-    log.flush()
+    log.info("Starting %d Spark workers and HDFS Datanodes.". inputs['numWorkers'])
```
Comment (fnothaft) [inline src/toil_scripts/adam_pipeline/spark_toil_script.py]: FIXME `.` instead of `,`

**Comment by fnothaft** [comment]
Found yonder a small type, added `needs work` tag.

**Comment by fnothaft** [comment]
Fixed the typo and removed the `needs work` tag.

File: src/toil_scripts/adam_pipeline/spark_toil_script.py [hunk 2]
```diff
@@ -40,7 +38,7 @@
 
 SPARK_MASTER_PORT = "7077"
 HDFS_MASTER_PORT = "8020"
-log = open("python.log", 'a')
+log = logging.getLogger(__name__)
```
Comment (hannes-ucsc) [inline src/toil_scripts/adam_pipeline/spark_toil_script.py]: Fuck yes! Thank you!

Comment (fnothaft) [inline src/toil_scripts/adam_pipeline/spark_toil_script.py]: Yes, this had been driving me batty as well.

**Pull request merged**